### PR TITLE
#211 (UNLV-FIA Homepage) Change Group to Club (Based On main)

### PR DESCRIPTION
**Pull request by Kelvyn-Meyers**
This pull request is for issue #211.

## CODE CHANGES
- Changed UNLV-FIA Group to Club
- Added a smiley face

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,8 +37,8 @@
 <div class="pimg_1">
   <div class="ptext">
     <div class="border">
-        <h1>UNLV-FIA</h1>
-        Group
+        <h1>UNLV-FIA :)</h1>
```
Comment (Lais3) [inline index.html:40]: Why is there a :)

**Review by Lais3 — CHANGES_REQUESTED** [review]
Review comments

**Review by Lais3 — APPROVED** [review]
Looks Good!

**Comment by Kelvyn-Meyers** [comment]
Closing example pull request

**Pull request closed without merging**